specs/DotNetProjectFile.Analyzers.Specs/IO/IOFile_specs.cs

```diff
@@ -0,0 +1,30 @@
+﻿using DotNetProjectFile.IO;
+
+namespace IO_specs;
+
+public class IO_File
+{
+    [TestCase("/", "./root/test/somefile.txt")]
+    [TestCase("\\", @".\root\test\somefile.txt")]
+    public void Supports_formatting(string format, string formatted)
+        => IOFile.Parse("./root/test/somefile.txt").ToString(format).Should().Be(formatted);
+
+    [Test]
+    public void Is_separator_agnostic()
+        => IOFile.Parse(".\\root\\test\\somefile.txt").Should().Be(IOFile.Parse("./root/test/somefile.txt"));
+
+    [TestCase(@"c:\Program Files\Microsoft.NET")]
+    public void Supports_type_conversion(IOFile path)
```

Comment:
Is the name of this test correct? Don't see any type conversion.

Reply:
That the Test case works, is because of that. ;)